src/Bootstrap/ButtonGroup.elm

```diff
@@ -57,12 +72,30 @@ type GroupItem msg
     = GroupItem (Html.Html msg)
 
 
-{-| Opaque type representing a button or link button, for composing button groups
+{-| Opaque type representing a button, for composing button groups
 -}
 type ButtonItem msg
     = ButtonItem (Html.Html msg)
 
 
+{-| Opaque type representing a link button, for composing button groups
+-}
+type LinkButtonItem msg
```

Comment:
I was going for consistency. 

I can't really think of cases where I'd mix links and buttons (would seem confusing to me, just as mixing radio and checkbox buttons is), but no good reason to disallow it either.